python/tk_multi_publish/publish.py

```diff
@@ -62,7 +62,12 @@ def work_template(self):
         """
         The current work file template as sourced from the parent app.
         """
-        return self._app.get_template("template_work")
+        template_name = "template_work"
+        template = self._app.get_template(template_name)
+        if template is None:
+            raise TankError("Failed to retrieve the '%s' template" % (template_name))
```

Comment:
This is great, but let's add a bit more info to the message to try to lead people down the right path:
```
"Failed to retrieve the '%s' template. This most likely means that the template is either not defined in the project's config, or is set to an improper value."
```